lib/Common/sensors_state.h

```diff
@@ -23,7 +23,7 @@ struct SensorState {
   float smoothedPumpFlow;
   float smoothedWeightFlow;
   float consideredFlow;
-  long pumpClicks;
+  float pumpClicks;
```

Comment:
This is mainly where the bug was. But I also killed some other potential rounding issues with how we were doing using setPumpToRawValue